public/app/features/explore/Logs.tsx

```diff
@@ -246,6 +256,11 @@ export class Logs extends PureComponent<Props, State> {
           timeZone={timeZone}
           getFieldLinks={getFieldLinks}
         />
+        {!loading && hasData && !scanning && displayMoreLogsBtn && (
+          <button className="gf-form-label gf-form-label--btn" onClick={showMoreOlderLogs}>
+            Show more logs
+          </button>
+        )}
```

Comment:
Button is being displayed when we don't load/scan data, there is only one active query (as for now) and query result is not empty.